tests/sdk/nodejs/examples/examples_test.go

```diff
@@ -148,7 +148,7 @@ func TestAccIngress(t *testing.T) {
 
 				integration.AssertHTTPResultWithRetry(t,
 					fmt.Sprintf("%s/index.html", stackInfo.Outputs["ingressIp"]),
-					nil, 5*time.Minute, func(body string) bool {
+					nil, 10*time.Minute, func(body string) bool {
```

Comment:
This seems unrelated to the other changes in the PR. Was it done to get the green light from CI, and included to avoid a more circuitous route through another PR?

Reply:
Yeah, that ingress test failed again, so I bumped the timeout to see if that would fix it. I'll pull this commit into a separate PR when I rework the message as suggested.

Reply:
Went ahead and merged this change as https://github.com/pulumi/pulumi-kubernetes/pull/2270

Reply:
Sorry I wasn't clear, I was OK with it being here -- but ➕ 💯  for pulling it out as its own thing anyway!